hudi-sync/hudi-hive-sync/src/main/java/org/apache/hudi/hive/HoodieHiveSyncClient.java

```diff
@@ -248,6 +250,20 @@ public Option<String> getLastCommitTimeSynced(String tableName) {
     }
   }
 
+  @Override
+  public Option<String> getMetastoreLocation(String tableName) {
+    try {
+      Table table = client.getTable(databaseName, tableName);
+      StorageDescriptor sd = table.getSd();
+      return Option.ofNullable(sd.getLocation());
+    } catch (NoSuchObjectException e) {
+      LOG.warn("the said table not found in hms " + tableId(databaseName, tableName));
+      return Option.empty();
+    } catch (Exception e) {
+      throw new HoodieHiveSyncException("Failed to get the metastore location from the table " + tableName, e);
+    }
+  }
```

Comment:
This is only used in the tests. Could you avoid adding the new API? Instead, you should use `client = Hive.get(config.getHiveConf()).getMSC()` to get the client in the test and then call this logic to get the table location and verify it.  The logic here should be moved to the test class.

Reply:
ok